src/main/java/org/galatea/starter/domain/IexHistoricalPricesDB.java

```diff
@@ -0,0 +1,131 @@
+package org.galatea.starter.domain;
+
+import java.math.BigDecimal;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+
+@Entity
+public class IexHistoricalPricesDB {
+
+  @Id
+  @GeneratedValue (strategy = GenerationType.AUTO)
+  private long id;
+
+  private String symbol;
+  private BigDecimal close;
+  private BigDecimal high;
+  private BigDecimal low;
+  private BigDecimal open;
+  private long volume;
+  private String date;
+
+  protected IexHistoricalPricesDB() {}
+
+/**
+ * Constructor method for IexHistoricalPrices.
+ * @param symbol Stock symbol
+ * @param close Price at close
+ * @param high Adjusted high price for the day
+ * @param low Adjusted low price for the day
+ * @param open Adjusted open price for the day
+ * @param volume Volume of trades for the day
+ * @param date Date queried (formatted yyyy-mm-dd)
+ */
+  public IexHistoricalPricesDB(final String symbol, final BigDecimal close, final BigDecimal high,
+      final BigDecimal low, final BigDecimal open, final long volume, final String date) {
+    this.symbol = symbol;
+    this.close = close;
+    this.high = high;
+    this.low = low;
+    this.open = open;
+    this.volume = volume;
+    this.date = date;
+  }
+
+  /**
+   * Constructs database entry from an IexHistoricalPrices object from API call
+   * @param entity an IexHistoricalPrices object
+   */
+  public IexHistoricalPricesDB(final IexHistoricalPrices entity) {
+    this.symbol = entity.getSymbol();
+    this.close = entity.getClose();
+    this.high = entity.getHigh();
+    this.low = entity.getLow();
+    this.open = entity.getOpen();
+    this.volume = entity.getVolume();
+    this.date = entity.getDate();
+  }
+
+  @Override
+  public String toString() {
+    return String.format(
+        "Entry[symbol='%s', date='%s', id=%d]",
+        symbol, date, id);
+  }
```

Comment:
Is this necessary? Lombok has `@ToString`

Reply:
Not anymore - I used it a bit for the logging done previously. I'll get rid of it.